### PR TITLE
add the link of Copernicus DEM file to the STAC item of HAND collection

### PR DESCRIPTION
**Pull request by cirrusasf**

### CODE REVIEW

**Comment by jacquelynsmale** [comment]
@cirrusasf I think the code looks good, but it's just missing a changelog entry! Do you have one ready to go?

**Comment by jtherrmann** [comment]
Could we update the title of the PR to be more descriptive? (What is the main purpose of the PR?)

**Comment by cirrusasf** [comment]
I like the comment from Jake. I would like to make the title of PR as "add link of DEM file to the hand stac item", but the tile of the PR is automatically picked from the comment of each commit. How to make the PR tile not change with each commit?

**Comment by jhkennedy** [comment]
@cirrusasf 
image

The title of the PR is up to you when you make the PR. GH suggests a title, but you can edit it before you click the create button. And it could be edited any time.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.3.2]
+### Added
+- Add the link of the Copernicus DEM file to the STAC item.
```
Comment (jhkennedy) [inline CHANGELOG.md:10]:
```suggestion
- Add a "related" link to the associated Cpernicus DEM tile for every HAND item

```

Comment (jacquelynsmale) [reply CHANGELOG.md:10]: Associate Copernicus  (fix spelling)

**Review by asjohnston-asf — DISMISSED** [review]

**Review by asjohnston-asf — APPROVED** [review]

**Comment by cirrusasf** [comment]
> @cirrusasf image
> 
> The title of the PR is up to you when you make the PR. GH suggests a title, but you can edit it before you click the create button. And it could be edited any time.

Thank you, Joe!

**Pull request merged**